examples/pytorch_lightning_simple.py

```diff
@@ -81,58 +68,80 @@ def forward(self, data):
 class LightningNet(pl.LightningModule):
     def __init__(self, trial):
         super(LightningNet, self).__init__()
-        self.model = Net(trial)
 
-    def forward(self, data):
+        # We optimize the number of layers, hidden units in each layer and dropouts.
+        n_layers = trial.suggest_int("n_layers", 1, 3)
+        dropout = trial.suggest_float("dropout", 0.2, 0.5)
+        output_dims = [
+            trial.suggest_int("n_units_l{}".format(i), 4, 128, log=True) for i in range(n_layers)
+        ]
+
+        self.model = Net(dropout, output_dims)
```

Comment:
I think we can define `self.model` using `torch.nn.Sequential`, which will be more concise.
Could you kindly do it?

Reply:
I thought for myself that it was better for the users to see that their model can be any `torch.nn.Module`. In Lightning we create model and experimenting with Trial.

Are you proposing to build a model inside Lightning?